packages/react/src/types.ts

```diff
@@ -168,4 +168,5 @@ export interface ProviderContextPrepared {
   target: Document
   theme: ThemePrepared
   originalThemes: (ThemeInput | undefined)[]
+  resolvedComponentVariables: Record<string, object>
```

Comment:
```suggestion
  DO_NOT_USE_OR_YOU_WILL_BE_FIRED_resolvedComponentVariables: Record<string, object>
```

Reply:
would `_internal_resolvedComponentVariables` be enough?